logstash-core/src/main/java/org/logstash/Event.java

```diff
@@ -226,13 +225,13 @@ public String toJson() throws JsonProcessingException {
         return result;
     }
 
-    public Map toMap() {
+    public Map<String, Object> toMap() {
         return Cloner.deep(this.data);
     }
 
     public Event overwrite(Event e) {
-        this.data = e.getData();
-        this.cancelled = e.isCancelled();
+        this.data = e.data;
+        this.cancelled = e.cancelled;
```

Comment:
curious to why this change?  Don't we want to encourage the standard getter pattern ? (is this a compiler warning ?)

Reply:
@jakelandis wops that randomly made it in here :) still, reduces the bytecode size of this method a little and since `Event` is `final` (-> getters provide no encapsulation in the context of `Event` itself) this is a fine cleanup imo :)